core/src/main/java/com/uber/m3/tally/BucketPairImpl.java

```diff
@@ -60,8 +58,6 @@ public BucketPairImpl(
             };
         }
 
-        Collections.sort(buckets);
```

Comment:
again, confirming this is fine because of the validate above?

Reply:
Yes

Reply:
This code is actually not used anymore